### PR TITLE
Add support for nv-raytracing-final

### PR DESCRIPTION
**Pull request by alelenv**
@johnkslang for review
Let me know if i missed anything else
Thanks

### CODE REVIEW

**Comment by s-perron** [comment]
The windows build that failed is complaining that `SpvOpTypeAccelerationStructureNV` is not defined.

**Comment by alelenv** [comment]
@s-perron  : Yes, i believe the order of upstreaming will be 
SPIRV-gitlab -> SPIRV-Headers -> SPIRV-Tools -> glslang -> shaderc
After which this should work

File: source/val/validate_builtins.cpp [hunk 1]
```diff
@@ -1688,11 +1688,12 @@ spv_result_t BuiltInsValidator::ValidatePrimitiveIdAtReference(
...
-        case SpvExecutionModelIntersectionNVX:
-        case SpvExecutionModelAnyHitNVX:
-        case SpvExecutionModelClosestHitNVX:
-        case SpvExecutionModelMissNVX: {
+        case SpvExecutionModelRayGenerationNV:
+        case SpvExecutionModelIntersectionNV:
+        case SpvExecutionModelAnyHitNV:
+        case SpvExecutionModelClosestHitNV:
+        case SpvExecutionModelMissNV:
+        case SpvExecutionModeCallableNV: {
```
Comment (dj2) [inline source/val/validate_builtins.cpp]: This should be SpvExecutionModelCallableNV

**Comment by dj2** [comment]
@alelenv Please note you need to sign the CLA before this can be accepted.

**Pull request merged**